mingw_cross_toolchain.cmake

```diff
@@ -1,9 +1,10 @@
 # CMake toolchain file, cf. README.mingw_cross
+
```

Comment:
why are there a bunch of new empty lines? could you clean that up?

Reply:
Readability! Looks more like other scripts I have seen. But I can adapt to your liking.

Reply:
done